lib/acts_as_paranoid/core.rb

```diff
@@ -163,10 +163,10 @@ def destroy_fully!
       end
     end
 
-    def destroy!
+    def destroy!(raise_error: true)
```

Comment:
I'd prefer a solution without adding an argument to `#destroy!`; ActiveRecord's version has no arguments.

How about making `#destroy` the main implementation, and letting [Rails' implementation](https://github.com/rails/rails/blob/83217025a171593547d1268651b446d3533e2019/activerecord/lib/active_record/persistence.rb#L554) handle the raising?

Reply:
@mvz good point, please review the changes, cheers